app/assets/javascripts/plot/highchartsops.coffee

```diff
@@ -92,7 +92,7 @@ class window.HighchartsOps extends PlotOps
     super(resize, reset, registerPen, resetPen, addPoint, updatePenMode, updatePenColor)
     @_chart              = Highcharts.chart(elemID, {})
     @_penNameToSeriesNum = {}
-
+    @_chart.options.exporting.buttons.contextButton.menuItems.pop()
```

Comment:
Hmm.  This is super-weird.  On my end, I'm actually seeing that popping once is even enough.  I need to do it twice to get rid of the duplicate entries.  Do you know why this is happening?

If it's justified—and given the small number of lines of code we had to add in order to get this functionality, I'm thinking that it very likely *is* justified in some way or another—we should add the second `pop` (or just drop the last two items with `slice`), and comment this.